src/schema/resolvers.js

```diff
@@ -255,21 +256,10 @@ module.exports = {
 
     syncInfo: async (root, { includeBalance }, { db: { Blocks } }) => {
       const fetchBalance = includeBalance || false;
-      let syncBlockNum = null;
-      let syncBlockTime = null;
-      let syncPercent = null;
-      let blocks;
-      try {
-        blocks = await Blocks.cfind({}).sort({ blockNum: -1 }).limit(1).exec();
-      } catch (err) {
-        logger.error(`Error query latest block from db: ${err.message}`);
-      }
-
-      if (blocks.length > 0) {
-        syncBlockNum = blocks[0].blockNum;
-        syncBlockTime = blocks[0].blockTime;
-        syncPercent = await calculateSyncPercent(syncBlockNum, syncBlockTime);
-      }
+      const syncBlockNum = Math.max(0, await blockchain.getBlockCount());
```

Comment:
if no blocks in block table, query rpc
if blocks, query db

Reply:
fixed good call